test/e2e/k8s_util.go

```diff
@@ -157,6 +157,26 @@ func (k *KubernetesUtils) getTCPv4SourcePortRangeFromPod(podNamespace, podNameLa
 	return int32(startPort), int32(endPort), nil
 }
 
+func ProbeCommand(url, protocol, wrapper string) []string {
```

Comment:
Maybe add comments for this function including what is `wrapper` used for.

Reply:
done